backend/src/apiserver/client_manager.go

```diff
@@ -51,6 +51,9 @@ const (
 
 	podNamespace          = "POD_NAMESPACE"
 	initConnectionTimeout = "InitConnectionTimeout"
+
+	visualizationServiceHost = "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST"
+	visualizationServicePort = "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT_HTTP"
```

Comment:
is _HTTP needed?

Reply:
Yes, see attached screenshot.
![image](https://user-images.githubusercontent.com/7911799/63458329-7ba90380-c407-11e9-9f5f-d6f795e1d4ab.png)

Reply:
can you use one without http suffix? for consistency.

Reply:
whoops yeah, I didn't realize that that it could be dropped.